src/networking/controllers/todos.ts

```diff
@@ -1,20 +1,52 @@
-import { Todo } from '../../types/globalTypes'
+import { ItemCheck } from '../../features/todosState'
+import { NewItem, Todo } from '../../types/globalTypes'
 import { httpService } from '../httpService'
-interface TodoResponse {
-  userId: number
-  id: number
+export interface TodoResponse {
+  description: string
+  id: string
   title: string
   completed: boolean
 }
 
+const server: string = '/tasks'
```

Comment:
What do you think of having an enum that tracks the endpoints of the app?

Maybe you can have in a different file (named for example `endpoints.ts`) an enum like this:

```js
enum Endpoints {
    GetTasks = '/tasks',
    CreateTask = '/tasks',
    ...
}
```